jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/MenusTests.java

```diff
@@ -210,7 +210,8 @@ public void menuAsPopoverMenuTests() {
         menuAsPopoverButton.click();
         activeMenu.is().displayed();
         activeMenu.has().numberOfOptions(3);
-        activeMenu.has().optionsTitles(OPTIONS_TITLES_NAMES);
+        //activeMenu.has().optionsTitles(OPTIONS_TITLES_NAMES);
```

Comment:
Comments are not allowed, keep only valid code

Reply:
Corrected